lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -78,10 +67,12 @@ if(CMAKE_UFE_V3_FEATURES_AVAILABLE)
             # See MayaUIInfoHandler.h comments.
             EditAsMayaCommand.cpp
             MayaUIInfoHandler.cpp
+            MayaUsdHierarchy.cpp
+            MayaUsdHierarchyHandler.cpp
+            MayaUsdRootChildHierarchy.cpp
```

Comment:
As discussed put these files here since they need the pulledinfo which requires path mapping which is Ufe v3.